lib/jira/http_error.rb

```diff
@@ -8,7 +8,7 @@ class HTTPError < StandardError
 
     def initialize(response)
       @response = response
-      @message = response.try(:message) || response.try(:body)
+      @message = response.try(:message).presence || response.try(:body)
```

Comment:
The response often has a message containing an empty string `''` while the body does contain usable information. Adding `.presence` will make sure the correct message is returned.